Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-getReadPreference.txt

MongoDB\\Collection::getReadPreference()
========================================

.. meta::

>    :description: Retrieve the read preference for a MongoDB collection using the `getReadPreference()` method.

Reviewer: ```suggestion
   :description: Retrieve the read preference of a collection with the MongoDB PHP Library.
```
